Void linux specific patch: do not clear screen in getty.

--- units/getty@.service.m4.orig	2012-07-02 18:53:11.843700293 +0200
+++ units/getty@.service.m4	2012-07-02 18:53:29.186280029 +0200
@@ -41,7 +41,7 @@ ConditionPathExists=/dev/tty0
 
 [Service]
 Environment=TERM=linux
-ExecStart=-/sbin/agetty %I 38400
+ExecStart=-/sbin/agetty --noclear %I 38400
 Type=idle
 Restart=always
 RestartSec=0
@@ -49,7 +49,7 @@ UtmpIdentifier=%I
 TTYPath=/dev/%I
 TTYReset=yes
 TTYVHangup=yes
-TTYVTDisallocate=yes
+#TTYVTDisallocate=yes
 KillMode=process
 IgnoreSIGPIPE=no
 
